tests/user_commands/tests.py

```diff
@@ -194,6 +194,24 @@ def test_call_command_unrecognized_option(self):
         with self.assertRaisesMessage(TypeError, msg):
             management.call_command('dance', unrecognized=1, unrecognized2=1)
 
+    def test_call_command_with_required_parameters(self):
+        """ Verify that all variations of required option passing produce the same result"""
+        out = StringIO()
+        management.call_command('required_option', "--need-me=asdf",
+                                "--need-me-too=fdsa", example='ex', verbosity=2, stdout=out)
+        out2 = StringIO()
+        management.call_command('required_option', need_me="asdf", needme2="fdsa",
+                                example="ex", verbosity=2, stdout=out2)
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).